rebuild-popular-crates.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
+# pup can be installed via: go get github.com/ericchiang/pup
+
+set -euo
```

Comment:
```suggestion
set -euo pipefail
```


-o pipefail is what you need to make the pipeline fail if pup or jq fails. Otherwise the pipeline will take the exit code of `sort`, I think.

Reply:
that is messing up the script actually. no output. didn't investigate too much.